corehq/apps/users/device_rate_limiter.py

```diff
@@ -63,10 +64,14 @@ def rate_limit_device(self, domain, user_id, device_id):
             )
             return False
 
-        metrics_counter(
-            'commcare.devices_per_user.rate_limit_exceeded', tags={'domain': domain, 'user_id': user_id}
-        )
-        return settings.ENABLE_DEVICE_RATE_LIMITER
+        if toggles.DEVICE_RATE_LIMITER.enabled(domain):
+            metrics_counter('commcare.devices_per_user.rate_limited', tags={'domain': domain, 'user_id': user_id})
+            return True
+        else:
+            metrics_counter(
+                'commcare.devices_per_user.rate_limited.test', tags={'domain': domain, 'user_id': user_id}
+            )
+            return False
```

Comment:
What do you think of putting the toggle value in a tag?
```suggestion
        enabled = toggles.DEVICE_RATE_LIMITER.enabled(domain):
        metrics_counter(
            'commcare.devices_per_user.rate_limited',
            tags={'domain': domain, 'user_id': user_id, 'enabled': str(enabled)},
        )
        return enabled
```
That would allow easy grouping in DD, but also combine the two metrics by default, which is admittedly maybe not what you want.

Reply:
Yeah I'm fine with this. Should still be able to do the separation in datadog regardless of a different metric vs tags. [8bbb387](https://github.com/dimagi/commcare-hq/pull/35781/commits/8bbb387a888396cd3858e34bb4ba5c040ff48a4a)